pkg/runner/command.go

```diff
@@ -153,8 +153,35 @@ func (cr *CommandRunner) handleInternalCmd() (int, string) {
 			return 1, fmt.Sprintf("openftp: Not enough information. %s", err.Error())
 		}
 
-		ftpClient := NewFtpClient(cr.data)
-		go ftpClient.RunFtpBackground()
+		sysProcAttr, err := demote(data.Username, data.Groupname)
+		if err != nil {
+			log.Debug().Err(err).Msg("Failed to get demote permission")
+
+			return 1, fmt.Sprintf("openftp: Failed to get demoted permission. %s", err.Error())
+		}
+
+		executable, err := os.Executable()
+		if err != nil {
+			log.Debug().Err(err).Msg("Failed to get executable path")
+
+			return 1, fmt.Sprintf("openftp: Failed to get executable path. %s", err.Error())
+		}
+
+		cmd := exec.Command(
+			executable,
+			"ftp",
+			data.URL,
+			data.HomeDirectory,
+		)
+		cmd.SysProcAttr = sysProcAttr
+		cmd.Stdout = os.Stdout
+		cmd.Stderr = os.Stderr
+
+		if err := cmd.Start(); err != nil {
```

Comment:
Please review if it's necessary to manage the child process with `Wait()` here.

Reply:
The purpose of `openftp` is to create a child process, but adding wait() would prevent the command from finishing until the child process ends. This seems inappropriate because the `command fin request` to `alpacon-server` would be sent after the child process terminates.
Instead, I added os.Exit(1) to the FtpClient.close() to terminate child process upon completion of the FtpClient connection.